x-pack/plugins/security_solution/common/search_strategy/security_solution/risk_score/common/index.ts

```diff
@@ -33,4 +33,7 @@ export enum RiskQueries {
   kpiRiskScore = 'kpiRiskScore',
 }
 
-export type RiskScoreAggByFields = 'host.name' | 'user.name';
+export enum RiskScoreEntity {
+  host = 'host',
+  user = 'user',
+}
```

Comment:
Could we turn this to `const enum`, we have had bundle size and performance problems using normal enums. I don't see any dynamic access (`RiskScoreEntity[x]`) so we should be able to change it.